docs/quickstart-traffic-splitting.md

```diff
@@ -48,35 +37,25 @@ Once the Service Mesh is installed:
         - '*'
         port:
           name: http
-          number: 80
+          number: 8
           protocol: HTTP
     EOF
     ```
+    
+1. Install the ODH-Model-Controller:
 
-## Install ODH-Model-Controller
-
-As noted in the prerequisites, make sure you install `modelmesh-serving` first.
-Then, install `odh-model-controller`:
-
-```shell
-# Clone the odh-model-controller repository
-git clone https://github.com/opendatahub-io/odh-model-controller.git
-# Enter into the cloned code path
-cd odh-model-controller
-# Deploy odh-model-controller
-make deploy -e IMG=quay.io/edgarhz/odh-model-controller:service-mesh-integration
-```
+    ```shell
+    # Clone the odh-model-controller repository
+    git clone https://github.com/opendatahub-io/odh-model-controller.git
 
-## Enable ODH Service Mesh features
+    # Enter into the cloned code path
+    cd odh-model-controller
 
-If
-you [installed ModelMesh with the quickstart instructions](https://github.com/opendatahub-io/modelmesh-serving/blob/main/docs/quickstart.md#1-install-modelmesh-serving)
-(as mentioned in the [Prerequisites](#prerequisites)) you will deploy models in
-the `modelmesh-serving` namespace, because this is the namespace where the
-ServingRuntimes are available.
+    # Deploy odh-model-controller
+    make deploy -e IMG=quay.io/edgarhz/odh-model-controller:service-mesh-integration
+    ```
 
-1. If you are using OSSM, enrol both the `opendatahub` and
-   this `modelmesh-serving` namespaces in the mesh:
+1. Enroll both the `opendatahub` and this `modelmesh-serving` namespaces in the mesh:
```

Comment:
This is only for OSSM. For Istio, this step is not done. So, I think we should keep the `If you are using OSSM,` text (or some more elegant writing)